src/argilla/client/feedback/dataset/remote/dataset.py

```diff
@@ -550,6 +553,13 @@ def metadata_properties(self) -> List["AllowedRemoteMetadataPropertyTypes"]:
         """
         return ArgillaMetadataPropertiesMixin.list(client=self._client, dataset_id=self.id)
 
+    @property
+    def vector_settings(self) -> List[RemoteVectorSettings]:
+        """Retrieves the `vector_settings` of the current dataset from Argilla"""
+        response = datasets_api_v1.list_vector_settings(client=self._client, id=self.id)
+
+        return [RemoteVectorSettings.from_orm(vector_settings) for vector_settings in response.parsed.items]
```

Comment:
In the rest of the remote schemas we're defining the `from_api` method instead of using the `from_orm` method, should we do the same here?

Reply:
Yes, I will use the same mechanism that we're using for the rest of the schemas. Anyway, maybe make sense to use the `from_orm` + [pydantic data bindings](https://docs.pydantic.dev/1.10/usage/models/#data-binding) which are commonly used for this kind of mapping. What do you think?

Reply:
Done